### DISCUSSION TITLE
Read query params to variables in Insights

### DISCUSSION THREAD

**Discussion by viters** [directus/directus#16793] [Feature Requests]
Similar to https://github.com/directus/directus/discussions/16776
It would be nice to allow settings variables using query params, so later we can craft links that lead to properly set up insights.

Consider again scenario with events, I would like to build insights for event and add button link on event details form that leads to insights for this very event.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
